client/src/models/Position.js

```diff
@@ -46,12 +46,13 @@ export default class Position extends Model {
       organization: yup
         .object()
         .nullable()
-        .default({})
+        .default(null)
+        .label("Organization")
         .test(
           "required-object",
           // eslint-disable-next-line no-template-curly-in-string
-          "${path} is required",
-          value => value && value.uuid
+          "${path} is a required field",
```

Comment:
Is the change in text needed?
To me at least, `Organization is required` is a better text than `Organization is a required field`. The user doesn't need to know what is the concept of a _field_ and etc...

Reply:
Position name was already displayed like that, so i made this change to make it consistent, i can revert this and make Position name as you want?